src/Helpers/SwapperV2.sol

```diff
@@ -8,9 +7,10 @@ import { LibAsset } from "../Libraries/LibAsset.sol";
 import { LibAllowList } from "../Libraries/LibAllowList.sol";
 import { ContractCallNotAllowed, NoSwapDataProvided, CumulativeSlippageTooHigh } from "../Errors/GenericErrors.sol";
 
-/// @title Swapper
+/// @title Swapper V2
 /// @author LI.FI (https://li.fi)
-/// @notice Abstract contract to provide swap functionality
+/// @notice Abstract contract to provide swap functionality with leftover token handling
+/// @custom:version 1.0.1
```

Comment:
This  change is more than just a bugfix, I would recommend to bump version to either 1.1.0 or even 2.0.0

Reply:
Fixed. https://github.com/lifinance/contracts/commit/b2e516c6